### PR TITLE
Final Milestone for JavaScript Capstone Project at development Branch

### PR DESCRIPTION
**Pull request by otmanTR**
We made the following changes in this branch:
-Used JavaScript 
-Used ES6 syntax.
-Used ES6 modules.
-Used callbacks and promises.
-Used liters.
-Used Webpack.
-Used Jest for testing
-Followed Gitflow.
-Sent and received data from an API.
-Used API documentation.
-Performed a code review for each team member.

### CODE REVIEW

File: src/__tests__/counterComment.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+describe("testing count comment", () => {
+
+
+test("Counting comments", () => {
+   let commentList = [];
+   commentList.push("BA");
+    
+    expect(countComment(commentList)).toBe(1) 
+})
+})
```
Comment (Ghiftee) [inline src/__tests__/counterComment.test.js]: - [ ] Good job testing this count function, however, kindly add more tests to cover all possible edge cases for example when there are zero comments.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @otmanTR  and @ambrose-kibet 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ You used gitflow
✔️ No linter errors
✔️ Your design is amazing
✔️ App runs as expected
However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

- [ ] As per the requirements, your app design should be similar in terms of data to the given mockup, kindly take a look at the following design differences and correct them

- ### Your footer design 
Screenshot 2023-02-17 at 11 56 32 (2)

- ### The given footer design
Screenshot 2023-02-17 at 12 02 16 (2)

- ### Your comment popup information section
Screenshot 2023-02-17 at 12 17 07 (2)

- ### The design's comment popup information section
Screenshot 2023-02-17 at 11 56 19 (2)


- [ ] The comment popup should not have a footer as per the design, kindly remove the footer from your comment popup

- ### Your comment popup footer
Screenshot 2023-02-17 at 11 55 25 (2)

- [ ] Kinldy put the total number of items displayed next to the writing as shown below

- ### Your design 
Screenshot 2023-02-17 at 12 02 22 (2)


- ### The given design
Screenshot 2023-02-17 at 11 56 38 (2)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- You can consider putting your live link demo in the section created for it in your README. This would make your README more professional

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/__tests__/counterComment.test.js (hunk 1 above)
Comment (Lizdev-05) [inline src/__tests__/counterComment.test.js]: Great job testing the counter 👍 . As the first reviewer said, kindly add more tests to cover all possible edge cases, I noticed you have not implemented this.

[OPTIONAL] I think you can mock the HTML so as to get an accurate test. ☺️ 🟢

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @otmanTR, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
- Correct GitHub Flow 👍

- Good commit history 💰

- Good PR title/description 💯

- All linters passed 🏅

- Personalised README file 👍
## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by danmainah — APPROVED** [review]
Hi @otmanTR and @ambrose-kibet , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello Team 👋,

_This is an additional review as a result of the second opinion request._

- I can see that the change requested has already been implemented and your PR approved 👍 

Keep rocking 🚀!
